webrtc-stats.html

```diff
@@ -976,7 +976,8 @@
              unsigned long long bytesSent;
              double             targetBitrate;
              unsigned long      framesEncoded;
-};</pre>
+             double             totalEncodeTime;
+          };</pre>
```

Comment:
we can do the indentation fix in a later PR, otherwise merge conflicts with existing PRs.